fhir-server/liberty-config/config/default/fhir-server-config-postgresql.json

```diff
@@ -59,7 +59,11 @@
             "datasources": {
                 "default": {
                     "type": "postgresql",
-                    "currentSchema": "fhirdata"
+                    "currentSchema": "fhirdata",
+                    "searchOptimizerOptions": {
+                        "from_collapse_limit": 16,
+                        "join_collapse_limit": 16
```

Comment:
Needs documentation on the new feature in the FHIRServerUsersGuide

Reply:
added